### PR TITLE
EOS-23588: Changes to accomodate miniprovisioning in kubernates envir…

### PR DESCRIPTION
**Pull request by rahul-tripathi-git**
# Problem Statement
- The mini provisioning needs modification to be performed in kubernetes environmenyt

# Design
-  The containerized deployment of cortx-py-utils needs to be accomodated in such a way that config phase will conatin most of the configuration changes. Also there should be no check for any service or any package presence

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [ ] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [ ] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [x] All the tests performed should be mentioned before Resolving a JIRA. 
- [x] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/src/setup/utils.py [hunk 1]
```diff
@@ -38,6 +39,28 @@ class Utils:
...
+            SetupError: If unable to delete file
+        """
+        for each_file in files:
+            if os.path.exists(each_file):
+                try:
+                    os.remove(each_file)
+                except OSError as e:
+                    raise SetupError(e.errno, "Error deleting file %s, \
+                        %s", each_file, e)
+        return 0
```
Comment (sachinpunadikar) [inline py-utils/src/setup/utils.py:62]: remove this return , not needed

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
work on the comment

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/setup/utils.py [hunk 2]
```diff
@@ -80,9 +79,9 @@ def _create_msg_bus_config(message_server_list: list, port_list: list, \
         config: dict):
         """ Create the config file required for message bus """
 
-        with open(r'/etc/cortx/message_bus.conf.sample', 'w+') as file:
+        with open(r'/etc/cortx/utils/message_bus.conf.sample', 'w+') as file:
```
Comment (rohit-k-dwivedi) [inline py-utils/src/setup/utils.py:82]: can have a common variable for the path `/etc/cortx/utils/`

**Review by rohit-k-dwivedi — APPROVED** [review]

**Pull request merged**